lms/djangoapps/coursewarehistoryextended/migrations/0001_initial.py

```diff
@@ -4,7 +4,7 @@
 from __future__ import absolute_import
 from django.db import migrations, models
 import django.db.models.deletion
-import coursewarehistoryextended.fields
+from lms.lib.fields import UnsignedBigIntAutoField
```

Comment:
you have to change the ref to `coursewarehistoryextended.fields.UnsignedBigIntAutoField` below to just `UnsignedBigIntAutoField`

Reply:
Thanks Alex!